contracts/cw20-base/src/contract.rs

```diff
@@ -47,13 +65,27 @@ pub fn instantiate(
         None => None,
     };
 
+    let marketing = match msg.marketing {
```

Comment:
Logo cannot be set on instantiate, correct, only later?

Reply:
This belongs in a Rustdoc (so the js devs see it as well)

Reply:
Yea, after all I am not sure why I did remove logo from instantiation. I though about not making the data big, but it would be big only if someone sends such logo so I don't see actual issue. I think I will bring it back in instantiation.